splunk_add_on_ucc_framework/__init__.py

```diff
@@ -654,28 +654,29 @@ def main():
         "--config",
         type=str,
         nargs='?',
-        help="Path to configuration file, Defaults to GlobalConfig.json in parent directory of source provided",
+        help="Path to configuration file, defaults to GlobalConfig.json in parent directory of source provided",
         default=None
     )
-    version = Version.from_git()
-    if not version.stage:
-        stage = 'R'
-    else:
-        stage = version.stage[:1]
-    
-    version_str = version.serialize(metadata=True,style=Style.SemVer)
-    version_splunk = f"{version.base}{stage}{version.commit}"
-    
     parser.add_argument(
         "--ta-version",
         type=str,
-        help="Version of TA, Deafult version is version specified in the package such as app.manifest, app.conf, and globalConfig.json",
-        default = version_splunk
+        help="Version of TA, default version is version specified in the package such as app.manifest, app.conf, and globalConfig.json",
+        default=None
     )
     args = parser.parse_args()
-    ta_version = args.ta_version.strip()
-    if not ta_version:
+    if not args.ta_version:
```

Comment:
We changed logic a little bit here. 
Before we stripped the passed ta-version and after that checked `if not ta_version`.

Don't know why that was that way, but let's assume that it makes sense

Reply:
We have 2 possible options with this `ta_version` argument:
 1. It is not specified - then `args.ta_version` is `None` - we use our Git-related logic.
 2. It is specified - then `args.ta_version` is not `None` - we strip it and use later.